virttest/utils_test/libvirt.py

```diff
@@ -3615,6 +3615,54 @@ def customize_libvirt_config(params,
     return obj_conf
 
 
+def remove_key_in_conf(value, conf_type="libvirtd",
+                       remote_params=None, restart_libvirt=False):
+    """
+    Remove settings in configuration file on local or remote
+    and restart libvirtd if needed
+
+    :param value: The value to delete
+    :param conf_type: The configuration type to update on localhost,
+        eg, "libvirt", "virtqemud"
+    :param remote_params: The params for remote access
+    :param restart_libvirt: True to restart libvirtd
+    :return: remote.RemoteFile object for remote file or
+        utils_config.LibvirtConfigCommon object for local configuration file
+
+    """
+
+    pattern_list = [value] if not isinstance(value, list) else value
```

Comment:
suggestion: sticking to same data type helps to avoid confusion, expect either a string or list from user itself, instead of typecasting to list here.

Reply:
Updated. Could you please help review again?